.gitattributes

```diff
@@ -0,0 +1 @@
+dist/* linguist-generated
```

Comment:
```suggestion
dist/ linguist-generated
```

It uses the same syntax as .gitignore which I think that implies only the direct children inside of the directory. I targeted the entire directory instead.

Reply:
@0x4007 Took the syntax from here:
https://github.com/github-linguist/linguist/blob/main/docs/overrides.md#generated-code

Reply:
According to the example, you should do double star

```suggestion
dist/** linguist-generated
```